lms/envs/common.py

```diff
@@ -2470,3 +2470,13 @@
 # Maximum number of retries per task for errors that are not related
 # to throttling.
 CREDIT_TASK_MAX_RETRIES = 5
+
+####################### In-line Analytics ######################
+INLINE_ANALYTICS_SUPPORTED_TYPES = {
+    'MultipleChoiceResponse': 'radio',
+    'ChoiceResponse': 'checkbox',
+    'OptionResponse': 'option',
+    'NumericalResponse': 'numerical',
+    'StringResponse': 'string',
+    'FormulaResponse': 'formula',
```

Comment:
This may be affected by Nick Parlante hinting pr, which I suspect will land soon. #6786